src/query/service/src/interpreters/interpreter_metrics.rs

```diff
@@ -0,0 +1,150 @@
+// Copyright 2022 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::time::Duration;
+use std::time::SystemTime;
+use std::time::UNIX_EPOCH;
+
+use common_exception::ErrorCode;
+use common_metrics::label_counter_with_val_and_labels;
+use common_metrics::label_histogram_with_val;
+
+use crate::sessions::QueryContext;
+use crate::sessions::TableContext;
+
+pub struct InterpreterMetrics;
+
+const QUERY_START: &str = "query_start";
```

Comment:
This file named `interpreter_metrics.rs`, but now almost metric in it is query_metrics. We will add more other kinds metrics in this file?